api/src/test/java/org/apache/iceberg/types/TestTypes.java

```diff
@@ -31,6 +31,8 @@ public void fromPrimitiveString() {
     assertThat(Types.fromPrimitiveString("BooLean")).isSameAs(Types.BooleanType.get());
 
     assertThat(Types.fromPrimitiveString("timestamp")).isSameAs(Types.TimestampType.withoutZone());
+    assertThat(Types.fromPrimitiveString("timestamp_ns"))
```

Comment:
this should also test `timestamptz_ns`

Reply:
Fixed in https://github.com/apache/iceberg/pull/9008/commits/c588d051236ba022ef291a4cbb0142ecef73755d. I've also added a test for `timestamptz`.